### PR TITLE
SSR-3219: install chromatic and add chromatic CI

### PR DESCRIPTION
**Pull request by voduchau**

### CODE REVIEW

File: .github/workflows/chromatic.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: 'Chromatic'
+
+on: push
+
+jobs:
+  chromatic-deployment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```
Comment (nmanhit) [inline .github/workflows/chromatic.yml:9]: Please use **actions/checkout@v3** if you can

File: package.json [hunk 2]
```diff
@@ -33,7 +33,8 @@
     "storybook": "start-storybook",
     "test": "rm -rf ./unit_test && tsc -p tsconfig.test.json && web-test-runner",
     "es-lint": "node_modules/.bin/eslint \"src/**\"",
-    "actionlint": "node-actionlint"
+    "actionlint": "node-actionlint",
+    "chromatic": "npx chromatic --project-token=141e2a8401fe"
```
Comment (nmanhit) [inline package.json:37]: Please remove this line, project-token is secret

File: .github/workflows/chromatic.yml (hunk 1 above)
Comment (voduchau) [reply .github/workflows/chromatic.yml]: I changed it to v3

File: package.json (hunk 2 above)
Comment (voduchau) [reply package.json]: I removed that command

**Pull request merged**